### ISSUE TITLE
docs @ "Best Practices: Web Assets" missing recommendation for location of non-public, asset source file

### ISSUE THREAD

**Issue by ghost** [symfony/symfony-docs#5199]
The Best Practices doc re: Web Assets

```
"Web Assets"
 http://symfony.com/doc/current/best_practices/web-assets.html
```

recommends

```
"Store your assets in the web/ directory."
```

and

```
"Keep in mind that web/ is a public directory and that anything stored here will be publicly accessible. For that reason, you should put your compiled web assets here, but not their source files (e.g. SASS files)."
```

But does NOT provide any recommendation for where those source files SHOULD go, consistent with the resto of the documentation

**Comment by xelaris** [comment]
I agree. The demo application stores those files in `app/Resources/assets` e.g. `app/Resources/assets/js` and `app/Resources/assets/sass`. This is the most consistent place IMO too. Would you open a PR?

**Comment by WouterJ** [comment]
I vote for `app/Resources/public`. The only problem is that Assetic and the `assets:install` command do not support this location (until we applied https://github.com/symfony/symfony-standard/issues/803).

/cc @javiereguiluz @weaverryan

**Comment by xelaris** [comment]
@WouterJ Since it's about **non-public** asset source files (like SASS), putting these files in `app/Resources/public` sounds strange to me. Neither these files are public there nor they should be published (without processing). So I think the name would be misleading.

**Comment by iltar** [comment]
Glad to see this issue has been raised already! Yesterday I decided that I wanted to finally move all "Resources" from my `App(Bundle)` to `/app/Resources`. I found that all public files were transferred via the `asset:install` which lead to the source files being in my web-dir because they were in `public`.

I've managed to get Assetics to load my stuff from `app/Resources/web/*` by changing the following configuration:

```yml
assetic:
    read_from: %kernel.root_dir%/Resources/web
    write_to: %kernel.root_dir%/../web
    bundles:
        - App
```

The problem here is that I still need to add App to bundles, otherwise it doesn't do anything. I had to modify `read_from` and `write_to` in order to detect the location and write to another, as by default it writes to the location it reads from...

I'm against using `/public` as recommendation because it's not public, if anything it's private. I'm not too satisfied with `/web` but currently I can't think of a more likely name. In the end it doesn't matter because no matter what processing scripts, libraries or bundles we use, it should all fit this recommendation.

/cc @WouterJ
